build/releaseBuild.yml

```diff
@@ -202,6 +202,14 @@ jobs:
       BuildDropPath: '$(Build.SourcesDirectory)\src'
       ManifestDirPath: '$(Build.SourcesDirectory)\artifacts'
 
+  - task: NuGetCommand@2
```

Comment:
nit: Should this be reverted? In MSAL we created a separate upload to VSTS pipeline so that we can control specific packages to upload to VSTS, as always uploading packages to the feed may accidentally upload packages that shouldn't be (like some packages for local dev testing or something).